L/libclangex/build_tarballs.jl

```diff
@@ -3,10 +3,15 @@
 using BinaryBuilder, Pkg
 using Base.BinaryPlatforms
 
-include("../../fancy_toys.jl")
+const YGGDRASIL_DIR = "../.."
+include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
+include(joinpath(YGGDRASIL_DIR, "platforms", "llvm.jl"))
 
 name = "libclangex"
-version = v"0.1.5"
+version = v"0.1.6"
+
+llvm_versions = [#=v"11.0.1",=# v"12.0.1", v"13.0.1", v"14.0.2"]
```

Comment:
I think we should build a new minor version of libclangex for each LLVM major version. We can add multi-version support in the wrapper library ClangCompiler.jl.

Reply:
For LLVMExtra we went the other way. LLVMExtra uses minor versions to denotate bug fixes and API changes, but it supports multiple version of LLVM at once. 
Same for Enzyme. This separates the API used by ClangCompiler from the implementation on different LLVM versions.

Different Minor version becomes difficult, since you need to select which version to install based on which LLVM version is loaded by Julia. We want to move away from the model where it is bound per Julia versions....